frontend/lib/src/components/widgets/ButtonGroup/ButtonGroup.tsx

```diff
@@ -323,10 +367,7 @@ function ButtonGroup(props: Readonly<Props>): ReactElement {
         }
         overrides={{
           Root: {
-            style: {
-              flexWrap: "wrap",
-              gap: gap,
-            },
+            style: getButtonGroupOverridesStyle(style, theme),
```

Comment:
it might make sense to memoize the calculated values and not call this function every time. I think we are not very stringent with the usage so far though.
@sfc-gh-bnisco Do you have an opinion on what our guideline here should be? Always memoize, only memoize if we believe this is an expensive function (here we do some object unpacking, which might be a tad slower than other kinds of calculations)